geneagrapher/grabber.py

```diff
@@ -24,8 +25,8 @@ def get_record(self, id):
         advisor ids, the mathematician name, the mathematician
         institution, and the year of the mathematician's degree.
         """
-        url = "http://genealogy.math.ndsu.nodak.edu/id.php?id=" + str(id)
```

Comment:
Note: the genealogy.math.ndsu.nodak.edu domain works again, but it's an alias to mathgenealogy.org.

```
$ host genealogy.math.ndsu.nodak.edu
genealogy.math.ndsu.nodak.edu is an alias for mathgenealogy.org.
mathgenealogy.org has address 66.42.119.162
mathgenealogy.org mail is handled by 0 mail.mathgenealogy.org.

$ host mathgenealogy.org
mathgenealogy.org has address 66.42.119.162
mathgenealogy.org mail is handled by 0 mail.mathgenealogy.org.
```